net/iperf3-mt/Makefile

```diff
@@ -0,0 +1,100 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+# Copyright (C) 2023 Jonas Jelonek
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=iperf
+PKG_VERSION:=iperf-3.15-mt-beta1
```

Comment:
```suggestion
PKG_VERSION:=3.15-mt-beta1
```

Reply:
Just this change doesn't work, I tried that before. The file that is downloaded from Github codeload seems to always have a name like `iperf-iperf-3.15-mt-beta1`, when extracted the directory will have the same name.
This means, the source is extracted to `iperf-iperf-3.15...` but the PKG_BUILD_DIR is generated by makefiles like `iperf-3.15...`, then it fails to build because it doesn't find the source.

Would you suggest to keep it as I did or overwrite PKG_BUILD_DIR?

Reply:
Yes, you should rewrite PKG_BUILD_DIR, that's common here, where it is as you said PKG_NAME-PKG_NAME-PKG_VERSION.

Reply:
done, thanks for the hint!